docs/docs/tutorials/creating-new-environments.md

```diff
@@ -1,181 +1,94 @@
-your environment’s `yaml` file, or it may not be visible in the list of available kernels.
+If you can "view" an environment, you can use it.
 :::
 
-## Step 3 - Installing packages via `conda` or `pip`
+## Select environments in editors
 
-To install new packages through either `conda` or `pip` you'll need to navigate back to the `Edit` page of your environment
-in the `conda-store` web interface.
+Instructions to select any environment you have access to in the following editing spaces:
 
-Adding `conda` packages to a `conda-store` environment via the command line, is not possible since the files are
-read-only in that context.
+* **JupyterLab** - In a Jupyter Notebook, click on the "Select Kernel" dropdown in the top-left corner, and select the environment.
 
-Additionally, adding `pip` packages via the command line is strongly discouraged. Not only do `conda` and `pip` not
-always play nice together, but it can lead to some inconsistencies across users' dependencies.
-This happens because when you `pip` install a package from the command line this is installed in your `.local` folder, so this changes will not be applied to other Nebari users.
+* **VS Code** - Click on the ⚙️ icon in the bottom-right to open `Settings` -> `Command Pallette`, and type "Python: Select Interpreter" and press <kbd>Enter</kbd> to get the list of environments to select from.
 
-:::note
-One exception to this rule is packages that you are actively developing.
-As you are building a package, you will likely want to install it as a _dev_ package.
-This can be done using:
-
-```bash
-pip install --no-build-isolation --user -e .
-```
+* **Terminal** - In the terminal window, you can use `conda` CLI commands like `conda activate <namespace>-<environment_name>` to activate the relevant environment and `conda env list` to view the list of available environments.
 
-Or, if you’re using `flit`, you can install with:
+## Special requirements
 
-```bash
-flit install -s
-```
+### Dask
 
-Please keep in mind that these are _NOT_ available to Dask workers!
-:::
+Include the [`nebari-dask` metapackage](https://anaconda.org/conda-forge/nebari-dask) in your environment to use Dask. This ensures you have the correct version of `dask-gateway` and the latest versions of `dask` and `distributed` libraries.
 
-## Troubleshooting
+By default, the `nebari-git-nebari-git-dask` environment (available to everyone) can be used for basic Dask workflows.
 
-1. If you have an environment that fails to build properly, you'll be able to see this failure on the build status page.
+### JHub App Launcher
 
-   Navigate to the `Full Logs` to investigate in more detail. Also, from the build status page you can trigger re-build in
-   case you hit issues with intermittent outages, etc.
+Include the `jhub-apps` package in your environment to create apps using the JHub App Launcher. You will also need the relevant app development framework and other necessary packages in the environment.
```

Comment:
You don't need jhub-apps itself. You need `jhsingle-native-proxy>=0.8.2` for all apps. 

For panel apps you'll also need `bokeh-root-cmd>=0.1.2` in additional to the panel deps.

Reply:
The app launcher now has its own docs which explain the environment requirements. I think its safe to remove this here.